chalicelib/responseobjects/elastic_request_builder.py

```diff
@@ -89,8 +88,7 @@ def create_query(filters):
         """
         # Each iteration will AND the contents of the list
         query_list = [Q('constant_score', filter=Q(
-            'terms', **{'{}__keyword'.format(
-                facet.replace(".", "__")): values['is']}))
+            'terms', **{facet: values['is']}))
```

Comment:
Is there a reason you took this out? If one of the translation is a path that looks like, in ES `path.to.nested.field`, I'm afraid this will break. Can you please take a look? There is now an ElasticSearch index with new and updated contents.